### PR TITLE
Add support for keyboard domains

### LINKED ISSUE #1914: add domain button to the keyboards
lots of users have asked us to add the most common domains (particularly for their country/language) to the keyboard.

TBD- where these will show up on the keyboards

All: .com .net .org . co
English .uk .us
French .fr
Italian .it
German .de
Japanese .jp
Korean .kr
Swedish .se
Norwegian .no
Danish .dk
Dutch .nl
Finnish .fi
Polish .pl
Russian - none
Spanish - none
Chinese (pinyin) - none
Chinese  (zhuyin) - none

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #1914 Add support for keyboard domains

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/PolishKeyboard.java [hunk 1]
```diff
@@ -65,4 +65,9 @@ public Locale getLocale() {
     public String getSpaceKeyText(String aComposingText) {
         return StringUtils.getStringByLocale(mContext, R.string.settings_language_polish, getLocale());
     }
+
+    @Override
+    public String[] getDomains(String... domains) {
+        return super.getDomains(".po");
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/PolishKeyboard.java]: This should be `.pl`

Comment (bluemarvin) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/PolishKeyboard.java]: Actually, lets get this in the RC. I'll file a follow up.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/GermanKeyboard.java [hunk 2]
```diff
@@ -63,4 +63,9 @@ public Locale getLocale() {
     public String getSpaceKeyText(String aComposingText) {
         return StringUtils.getStringByLocale(mContext, R.string.settings_language_german, getLocale());
     }
+
+    @Override
+    public String[] getDomains(String... domains) {
+        return super.getDomains(".de");
```
Comment (philip-lamb) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/GermanKeyboard.java]: German keyboard would also be used in Austria and Switzerland, so this should have included `.at` and `.ch`.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/SpanishKeyboard.java [hunk 3]
```diff
@@ -48,4 +48,9 @@ public Locale getLocale() {
     public String getSpaceKeyText(String aComposingText) {
         return StringUtils.getStringByLocale(mContext, R.string.settings_language_spanish, getLocale());
     }
+
+    @Override
+    public String[] getDomains(String... domains) {
+        return super.getDomains(".es");
```
Comment (philip-lamb) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/SpanishKeyboard.java]: Spanish keyboard also applies across Latin America.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/GermanKeyboard.java (hunk 2 above)
Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/GermanKeyboard.java]: Thanks @philip-lamb Added [here](https://github.com/MozillaReality/FirefoxReality/issues/2132).

File: app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/SpanishKeyboard.java (hunk 3 above)
Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/SpanishKeyboard.java]: Also added [here](https://github.com/MozillaReality/FirefoxReality/issues/2132)

**Comment by jvonitter** [comment]
Spanish wasn't supposed to have any special domains (see #1914) because so many different countries use it. I don't mind if y'all feel strongly about including .es but there was a reason I left it off.

**Pull request merged**